docs/includes/elements/container.md

```diff
@@ -74,3 +76,149 @@ Another aspect is the new elements hierarchy. We needed to decide which elements
       nested Containers.
 - Some of the styles in the `_container.scss` file are leftovers from the POC phase that should be editor-only, and
   should be moved to another file.
+
+## How-Tos:
+
+### Adding a new control:
+
+In the `container.php` file, add a new control as any other widget:
+
+```PHP
+// container.php
+
+$this->add_control(
+	'overflow',
+	[
+		'label' => esc_html__( 'Overflow', 'elementor' ),
+		'type' => Controls_Manager::SELECT,
+		'default' => '',
+		'options' => [
+			'' => esc_html__( 'Default', 'elementor' ),
+			'hidden' => esc_html__( 'Hidden', 'elementor' ),
+			'auto' => esc_html__( 'Auto', 'elementor' ),
+		],
+		'selectors' => [
+			'{{WRAPPER}}' => '--overflow: {{VALUE}}',
+		],
+	]
+);
+```
+
+Notice that we've set a CSS variable to the `{{WRAPPER}}` - We're gonna use that in the SCSS file:
+
+```SCSS
+// _container.scss
+
+// First, reset the CSS var in the top of the file, to its initial value:
+--overflow: visible;
+// ...
+// other vars declarations
+// ...
+
+// Then, consume it:
+overflow: var( --overflow );
+```
+
+This variables reset method is required in order to avoid style leakage as stated above.
+
+#### Guidelines:
+
+- Name the CSS variable after the corresponding CSS property.
+- Try to always use `{{WRAPPER}}` as the selector, and consume the variable under the proper selector in the SCSS file
+  instead of putting a long selector in the PHP file. 
+- Make sure the default value you set for the property is the real default according to 
```

Comment:
If the element has a default value (for example, align=left) it should be used for the CSS variable initial value.

Reply:
Yup, done.